### DISCUSSION TITLE
How to get all style property name and value for selected component?

### DISCUSSION THREAD

**Discussion by san-123** [GrapesJS/grapesjs#3838] [Q&A]
Hi artf, 

Actually I drag and drop textbox then  I applied some styles to it  from stylemanager. 

My question is how to get all style  property name and value for selected component (textbox)?

Pl guide me.

**Comment by ronaldohoch** [comment]
Hello!
I think this can help you.

`editor.getSelected().rule.attributes.style`

![image](https://user-images.githubusercontent.com/2287371/135660202-11a4a7e2-a8a0-436b-b08e-79418e558f3f.png)
